projects/plugins/jetpack/modules/tiled-gallery/math/class-constrained-array-rounding.php

```diff
@@ -79,36 +84,11 @@ private static function adjust_constrained_array( &$bound_array_int, $adjustment
 			++$bound_array_int[ $i % $length ]['floor'];
 		}
 
-		usort( $bound_array_int, array( 'self', 'cmp_asc_index' ) );
-	}
-
-	/**
-	 * Compare fraction values of two arrays.
-	 *
-	 * @param array $a - the first array we're comparing.
-	 * @param array $b - the second array we're comparing.
-	 *
-	 * @return int
-	 */
-	private static function cmp_desc_fraction( $a, $b ) {
-		if ( $a['fraction'] === $b['fraction'] ) {
-			return 0;
-		}
-		return $a['fraction'] > $b['fraction'] ? -1 : 1;
-	}
-
-	/**
-	 * Compare index values of two arrays.
-	 *
-	 * @param array $a - the first array.
-	 * @param array $b - the second array.
-	 *
-	 * @return int
-	 */
-	private static function cmp_asc_index( $a, $b ) {
-		if ( $a['index'] === $b['index'] ) {
-			return 0;
-		}
-		return $a['index'] < $b['index'] ? -1 : 1;
+		usort(
+			$bound_array_int,
+			static function ( $a, $b ) {
+				return strcmp( $a['index'], $b['index'] );
```

Comment:
This one is less clear, but probably also a number as it's derived from the key in the original array.
https://github.com/Automattic/jetpack/blob/fb86ccacc90763a4d0e10c9fd02275a88288f3cf/projects/plugins/jetpack/modules/tiled-gallery/math/class-constrained-array-rounding.php#L58

Reply:
Done in b503fca7d2b44b8c41de15263072a4145cc1b69c